app/models/invoice.rb

```diff
@@ -6,4 +6,8 @@ class Invoice < ApplicationRecord
   has_many :merchants, through: :items
 
   enum :status, ["cancelled", "in progress", "completed"]
-end
\ No newline at end of file
+
+  def self.unshipped_invoices
+    Invoice.joins(:invoice_items).where("invoice_items.status != 2").group("invoices.id").order(:created_at)
```

Comment:
Nice! I like how y'all chose invoice items != to 2 as opposed to either equaling 0 or 1, makes it a lot cleaner